src/solver/application.cpp

```diff
@@ -281,8 +281,11 @@ void Application::processCaption(const Yuni::String& caption)
 void Application::prepareWriter(Antares::Data::Study& study,
                                 Benchmarking::IDurationCollector& duration_collector)
 {
+    ioQueueService = std::make_shared<Yuni::Job::QueueService>();
```

Comment:
@flomnes : Note that the zip writing is now handled by this separate thread pool which is allocated only 1 thread.

Since I don't do "introspection" on the type of writer used, the thread is created also when using plain files, and not used, which was not the case before.
The runtime negligible cost seems reasonable to me.

Reply:
You could use `IResultWriter::needsTheJobQueue()`, or even make `EnsureQueueStartedIfNeeded` public and use it.

Reply:
The writer is not yet created when we need to know this:
we can instead have a factory class (`ResultWriterFactory`) which provides that information before creation of the writer, but honestly I think this is overkill, having a sleeping thread is really not an issue.

It would make responsibilities clearer to leave it as is: the application provides a usable thread pool, the writer has the choice to use it or not.
Actually, I would also be in favor of removing `EnsureQueueStartedIfNeeded` completely, together with the `needsTheJobQueue()` in the interface, to make things simpler. 
It's the responsibility of the application to provide a working thread pool, the writer should not need to worry about starting anything.